src/components/Heading/Heading.test.tsx

```diff
@@ -158,6 +180,40 @@ describe("Heading", () => {
     const withOtherProps = renderer
       .create(<Heading id="props" text="Heading" data-testid="props" />)
       .toJSON();
+    const withOverline = renderer
```

Comment:
Thanks for adding these. One last addition, can you add an a11y test with either or both of these prop values passed? The original heading DOM structured is tested but not the one with the `hgroup` element.